winui/Scheduler/Month-View.md

```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
 
 ![customize-leading-and-trailing-days-visibility-in-winui-scheduler-month-view](Month-View_Imges/customize-leading-and-trailing-days-visibility-in-winui-scheduler.png)
 
+## Customizing the Number of Visible Weeks
+The number of week visible in the month view can be changed by setting the NumberOfVisibleWeeks property of the MonthViewSettings in SfScheduler. The default value is 6, and the valid range is from 1 to 6. Even if leading and trailing days are hidden, they will appear when changing the number of visible weeks, as no additional days exist during this change.
+
+{% tabs %}
+{% highlight xaml %}
+<scheduler:SfScheduler x:Name="Schedule" 
+                     ViewType="Month">
+    <scheduler:SfScheduler.MonthViewSettings>
+        <scheduler:MonthViewSetting NumberOfVisibleWeeks="3"/>
+    </scheduler:SfScheduler.MonthViewSettings>
+</scheduler:SfScheduler>
+{% endhighlight %}
+{% highlight c#%}
+this.Schedule.ViewType = SchedulerViewType.Month;
+this.Schedule.MonthViewSettings.NumberOfVisibleWeeks = 3;
+{% endhighlight %}
+{% endtabs %}
+
```

Comment:
Add notes:

- The week number range is limited to values between 1 and 6. Any value outside this range will cause the `NumberOfVisibleWeeks` to default to 6.
- The `NumberOfVisibleWeeks` property dynamically controls the number of weeks displayed in the scheduler's month view.
- The `DisplayDate` property defines the first row of dates shown in the scheduler's month view.

Reply:
Added notes.